### ISSUE TITLE
Is proxy actually being used?

### ISSUE THREAD

**Issue by AndrewPetrovics** [distubejs/ytdl-core#126]
## Describe the bug

I am using proxies. When testing locally everything works fine, but after deploying to my server I start getting 403s. I don't understand why there's a difference if I'm using a proxy. Any ideas?

Here is my code...
```js
const agent = ytdldistube.createProxyAgent({ uri: "My proxy url" });
const info = await ytdldistube.getInfo(videoUrl, { agent, })

ytdldistube.downloadFromInfo(info, { agent })
        .on("error", (err) => {
          // I get a 403 error after deploying to my server, but no error when running locally
        })
        .pipe(outputStream)
```

## Environment

- @distube/ytdl-core version: 4.15.1
- Node.js version: v21.5.0
- Operating system: Linux

**Comment by ugabb** [comment]
Same here

**Comment by faizur9** [comment]
same here on server not working

**Comment by skick1234** [comment]
Fixed with #128
